LICENSE

```diff
@@ -1,3 +1,26 @@
+radicle-upstream is Copyright © 2019-2020 Monadic GmbH <company@monadic.xyz>
+
+radicle-upstream is licensed under version 3 of the GNU General Public License (GPLv3) with the Radicle Linking Exception. Any contribution to radicle-upstream must be made subject to the developer certificate of origin (DCO). 
```

Comment:
I'd expect this file to be hard-wrapped.

Reply:
Also wonder whether we should not use the same LICENSE file everywhere:
https://github.com/radicle-dev/radicle-bins/pull/53/files \cc @cloudhead

Reply:
fixed. we are following Ange's instructions on this one, so I would be hesitant to diverge.